### PR TITLE
DDP-5667: Additional Info Labels

### PR DESCRIPTION
**Pull request by EdPokhvalin**
This change is [Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/476)

### CODE REVIEW

**Review by MocanaAtBroad — COMMENTED** [review]
Did we forget about displaying additionalInfoFooter?

> *[Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/476#-:-MW062kJ9FUZmyJdWqY3:b-76811z)* status: 0 of 13 files reviewed, all discussions resolved (waiting on @alextanlan and @fat23cat)

**Review by EdPokhvalin — COMMENTED** [review]
Yep, I just forgot to push that commit

> *[Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/476#-:-MW44QW97CQq5COzNLtL:bv9xo3g)* status: 0 of 15 files reviewed, all discussions resolved (waiting on @alextanlan and @fat23cat)

File: ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts [hunk 1]
```diff
@@ -49,7 +49,12 @@ import { BlockVisibility } from '../../../models/activity/blockVisibility';
                                    [block]="block"
                                    [readonly]="readonly"
                                    (valueChanged)="onChange($event)">
-    </ddp-activity-agreement-answer>`
+    </ddp-activity-agreement-answer>
+    <span *ngIf="block.additionalInfoFooter && block.shown"
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts]: I noticed that every block/component into the template has ` && block.shown` in their condition.
What do you think is it not worth extracting it and moving into the component level, like: 
```
 <ng-container *ngIf="block.shown">
      <ddp-activity-boolean-answer *ngIf="isBooleanQuestion(block)"></>
      <ddp-activity-text-answer *ngIf="isTextQuestion(block)"></>

```      etc...

**Review by EdPokhvalin — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/broadinstitute/ddp-angular/476)* status: 0 of 15 files reviewed, 1 unresolved discussion (waiting on @alextanlan and @fat23cat)

___
*[ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts, line 53 at r2](https://reviewable.io/reviews/broadinstitute/ddp-angular/476#-MW451XLX_3lgUgdxVfR-r2-53:-MW9FSIO0HYhLCKZStE4:b-fk46qv) ([raw file](https://github.com/broadinstitute/ddp-angular/blob/5b8360e26562ab03e036d08ecc49c608859b7e80/ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts#L53)):*
<details><summary><i>Previously, alextanlan (Alex Maratkanov) wrote…</i></summary><blockquote>

I noticed that every block/component into the template has ` && block.shown` in their condition.
What do you think is it not worth extracting it and moving into the component level, like: 
```
 <ng-container *ngIf="block.shown">
      <ddp-activity-boolean-answer *ngIf="isBooleanQuestion(block)"></>
      <ddp-activity-text-answer *ngIf="isTextQuestion(block)"></>

```      etc...      

</blockquote></details>

It's a good idea, I fixed it


<!-- Sent from Reviewable.io -->

**Review by alextanlan — APPROVED** [review]

**Pull request merged**